spec/todo_lint/options_spec.rb

```diff
@@ -0,0 +1,60 @@
+require "spec_helper"
+
+module TodoLint #:nodoc:
+  describe Cli do
+    let(:fake_project_path) { File.expand_path("../../fake_project", __FILE__) }
+    let(:lib_path) { File.expand_path("../../../lib", __FILE__) }
+    let(:app_rb) { File.join(fake_project_path, "app.rb") }
+    let(:app_spec_rb) { File.join(fake_project_path, "spec", "app_spec.rb") }
+    let(:app_js) { File.join(fake_project_path, "app.js") }
+    let(:app_coffee) { File.join(fake_project_path, "app.coffee") }
+    let(:todo_lint_rb) { File.join(lib_path, "todo_lint.rb") }
+    it "should not look at excluded files, and look at .rb by default" do
+      cli_test = Cli.new(["todo_lint", "-e", "[#{app_rb}]"])
+      options_hash = cli_test.instance_variable_get(:@options)
```

Comment:
nice

if you find yourself wanting to assert on the value of these, it would be kind of fine to add a public `attr_reader` for accessing these in a more traditional way